### PR TITLE
Unit-tests-jest - Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**

### CODE REVIEW

File: unit-tests-jest/numbers.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `$${amount}.00`;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.ts:12]: This does't work for decimal amounts.

File: unit-tests-jest/numbers.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+): Record<string, unknown> {
+  const result: Record<string, unknown> = {};
+  Object.entries(obj).forEach(([key, value]) => {
+    if (typeof value === 'number') {
+      result[key] = value * multiplier + 1;
+    } else {
+      result[key] = value;
+    }
+  });
+  return result;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.ts]: This doesn't modify the original object, which the description says it should do.

**Review by RobertGardner — APPROVED** [review]
You haven't covered edge cases in your tests, which is why you missed a couple of bugs. But what you have done is good and correct.
